packages/front-end/components/FactTables/FactMetricModal.tsx

```diff
@@ -883,11 +923,13 @@ export default function FactMetricModal({
             >
               <Tab id="query" display="Query Settings">
                 <MetricDelayHours form={form} />
-                <MetricCappingSettingsForm
-                  form={form}
-                  datasourceType={selectedDataSource.type}
-                  metricType={type}
-                />
+                {type !== "quantile" ? (
+                  <MetricCappingSettingsForm
+                    form={form}
+                    datasourceType={selectedDataSource.type}
+                    metricType={type}
+                  />
+                ) : null}
```

Comment:
Are we also protecting against this in the back-end or when generating SQL?  If I start with a mean metric, enable capping, and then switch the type to quantile, the capping settings will be saved in Mongo.

Reply:
I had a TODO that I will remove and action on!